lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -346,9 +376,9 @@ sub add_batch_job {
 }
 
 sub select_unfinished_tests {
-    my ( $self ) = @_;
+    my ( $self, $queue, $test_run_timeout, $test_run_max_retries ) = @_;
```

Comment:
here as well

Reply:
I've replaced all occurrances of `$queue` with `$queue_label`.